src/components/faqTemplates/shared/ExpandableFaq.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
```

Comment:
Why the `shared` subfolder?

Reply:
It's for components that may be useful for other templates under faq

Reply:
It was confusing to me to have all the files flat